packages/ra-input-rich-text/src/index.js

```diff
@@ -61,6 +61,10 @@ class RichTextInput extends Component {
         this.quill = null;
     }
 
+    componentWillReceiveProps(nextProps){
```

Comment:
I'm not convinced this solves a bug. Please remove this change and open a new PR for it, where we can choose to merge it or leave it independently of the other issue.

Reply:
hi @fzaninotto , I will remove this commit and open a new PR with it. Thanks Regards